src/style.css

```diff
@@ -1 +1,33 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star:last-child {
+  margin-bottom: 0;
+}
+
+.stars__star {
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  flex-shrink: 0;
```

Comment:
Do you really need this property? Also, pay attention to the CSS properties order
https://9elements.com/css-rule-order/

Reply:
And really, thank you!